www/src/pages/contribute.md

```diff
@@ -57,4 +98,4 @@ If you'd like to have an impact on this specification but don't have the privile
 
 ## Code of conduct
 
-- [Code of conduct](https://github.com/design-tokens/community-group/blob/main/CODE_OF_CONDUCT.md)
+- [Code of conduct](https://github.com/design-tokens/community-group/blob/main/governance/CODE_OF_CONDUCT.md)
```

Comment:
Minor request: this is more of an open source thing but CODE_OF_CONDUCT.md is a “standard” file (alongside LICENSE and README) that is more visible in the root. I think having other things in `./governance/` makes sense but I do think it’s important that folks see CODE_OF_CONDUCT easily (and more importantly, read it!)